js/script.js

```diff
@@ -0,0 +1,16 @@
+document.addEventListener('DOMContentLoaded', function() {
+
+let content = document.getElementById("circle");
+let button = document.getElementById("button");
+
+button.onclick = function() {
```

Comment:
Before you start listening on click, make sure the `button` and `content` elements do exist itself. You can do simple checking like:

```
if (!button || !content) {
   console.warn('Unable to read required HTML tags.');
   return;
}
```